### ISSUE TITLE
demo code has an IndentationError 

### ISSUE THREAD

**Issue by zhangming0509** [gabrielfalcao/lettuce#514]
the last demo in the [lettuce tutorial](http://lettuce.it/tutorial/simple.html#tutorial-simple) has an IndentationError 

```
  def have_the_number(self, step, number):
    '''I have the number (\d+)'''
      self.set_number(number)
```

**Comment by nikolas** [comment]
good point, looks like the problem can be fixed here: https://github.com/gabrielfalcao/lettuce/blob/master/_public/tutorial/simple.html#L412

But that file is complicated.. I can't figure it out.
